lms/routes.py

```diff
@@ -250,6 +250,11 @@ def includeme(config):  # noqa: PLR0915
         factory="lms.resources.dashboard.DashboardResource",
     )
 
+    config.add_route(
+        "api.dashboard.organizations.courses",
+        "/api/dashboard/organizations/{public_id}/courses",
```

Comment:
Can we call the param `organization_id` instead of `public_id`? It will make it more clear when used out of the context of the URL itself.

Reply:
Went for `organization_public_id` to distinguish  it from the PK.

Added a note to the board about using this naming in all routes.